lib/workers/pr/changelog/index.js

```diff
@@ -2,26 +2,46 @@ const { logger } = require('../../../logger');
 const versioning = require('../../../versioning');
 const sourceGithub = require('./source-github');
 const { getReleases } = require('./releases');
+const sourceGitlab = require('./source-gitlab');
+const hostRules = require('../../../util/host-rules');
+const util = require('util');
+const URL = require('url');
 
 module.exports = {
   getChangeLogJSON,
 };
 
 async function getChangeLogJSON(args) {
-  const { sourceUrl, versionScheme, fromVersion, toVersion } = args;
+  logger.debug(
```

Comment:
Too much debug has been added here. This function is called a lot, and would add significantly to the log storage. Consider logger.trace() instead, but I suspect these can mostly be removed.

Reply:
The majority were used for development-time testing and can be removed, yes.